legwork/snr.py

```diff
@@ -4,13 +4,45 @@
 from legwork import strain, psd, utils, evol
 import astropy.units as u
 
-__all__ = ['snr_circ_stationary', 'snr_ecc_stationary', 'snr_circ_evolving',
-           'snr_ecc_evolving']
+__all__ = ['amplitude_modulation', 'snr_circ_stationary', 'snr_ecc_stationary',
+           'snr_circ_evolving', 'snr_ecc_evolving']
+
+
+def amplitude_modulation(theta, phi, psi, inc):
```

Comment:
I think this would probably be cleaner in the strain module as you mentioned in your description.

If you *don't* do that though then I think you should put the condition for checking if everything is `None` directly in the function and just return `1.0` to save the repeated conditions in the rest of the module.

Reply:
this is done!